internal/rpcserver/rpcserver.go

```diff
@@ -147,7 +147,7 @@ var (
 	zeroHash chainhash.Hash
 )
 
-// Errors
+// Errors.
```

Comment:
This should just be removed imo.  Each individual error is commented, so there is no need for an uninformative block comment.

Reply:
Done